packages/react-components/react-text/src/components/presets/Body1Strong/Body1Strong.test.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { Body1Strong } from './Body1Strong';
+import { isConformant } from '../../../testing/isConformant';
+
+describe('Body1Strong', () => {
+  isConformant({
+    Component: Body1Strong,
+    displayName: 'Body1Strong',
+  });
+
+  it('renders a default state', () => {
+    const result = render(<Body1Strong>Default Body1Strong</Body1Strong>);
+    expect(result.container).toMatchSnapshot();
```

Comment:
can we use `toMatchInlineSnapshot()`, I know other teams are using this pattern, but I was advising to not use it as it creates unnecessary indirection and makes the test black magic.

Reply:
I reused the tests we had and copied over to all new presets, but that makes more sense. Updated!